.github/workflows/ci.yml

```
@@ -160,7 +160,7 @@ jobs:
         include:
           - name: Test Linux py38
             os: ubuntu-latest
-            pyversion: '3.8'
+            pyversion: '3.9'
```

Let's match the version with the title - until we remove this soon :)
```suggestion
            pyversion: '3.8'
```